### DISCUSSION TITLE
Support multiple types of expiry for one post

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-future#352] [Feature Requests]
> I would like Publishpress to not only set the post to draft on the specified date, but also move it to a different category

**Comment by stevejburge** [comment]
> What I'm missing now is combining conditions. It would be great if users could replace/add/remove post category but also tags at the same time. So it's about setting two actions to be performed by cron on the expiration date of the post.

**Comment by andergmartins** [comment]
Another option is allowing multiple actions for the same post on different dates, not only in a sequence at the same time. We can't schedule another action for the post is has already another one pending.

**Comment by stevejburge** [comment]
One more request for this today

**Comment by andergmartins** [comment]
Related to #341

**Comment by andergmartins** [comment]
Released in the Pro version as part of the Workflow Builder, on 3.4.0
